ArjunaJTS/jts/classes/com/arjuna/ats/internal/jts/orbspecific/ibmorb/recoverycoordinators/JavaIdlRCManager.java

```diff
@@ -170,12 +170,10 @@ private synchronized void initialise ()
        * want to expose the setting via the public EnvironmentBean config.
        */
         String env = System.getProperty("com.arjuna.ats.internal.jts.orbspecific.javaidl.recoverycoordinators.noDaemon");
-        String env2 = System.getProperty("com.arjuna.ats.internal.jts.orbspecific.jacorb.recoverycoordinators.noDaemon");
-        String env3 = System.getProperty("com.arjuna.ats.internal.jts.orbspecific.ibmorb.recoverycoordinators.noDaemon");
+        String env2 = System.getProperty("com.arjuna.ats.internal.jts.orbspecific.ibmorb.recoverycoordinators.noDaemon");
```

Comment:
Similar to the rename of okX -> X_orb_tests_ok in scripts/hudson/narayana.sh elsewhere in this pull, perhaps we should make more descriptive names here? (or keep env3 for imborb if it's possible and contiguous is not required?)